Invalid module path in botbuilder-ai

## Versions
botbuilder-ai 4.5.0  
Linux

## Describe the bug
> \> tsc
> node_modules/botbuilder-ai/lib/qnamaker-interfaces/index.d.ts:12:15 - error TS2307: Cannot find module './qnamakerTraceinfo'.
> 12 export * from './qnamakerTraceinfo';

## To Reproduce
Steps to reproduce the behavior:
1. Write code using botbuilder-ai.  
    e.g. `import * as appInsights from 'botbuilder-applicationinsights';`
2. execute `tsc`
3. See error

[bug]

--- a/libraries/botbuilder-ai/src/qnamaker-interfaces/index.ts
+++ b/libraries/botbuilder-ai/src/qnamaker-interfaces/index.ts
@@ -10,6 +10,6 @@ export * from './qnamakerEndpoint';
 export * from './qnamakerMetadata';
 export * from './qnamakerOptions';
 export * from './qnamakerResult';
-export * from './qnamakerTraceinfo';
+export * from './qnamakerTraceInfo';
 export * from './feedbackRecord';
 export * from './feedbackRecords';

